src/components/stepforms/DissolveStepForm.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <StepFormHeader
+      :title="title"
+      :stepName="editedStep.name"
+      :version="version"
+      :backendError="backendError"
+    />
+    <MultiselectWidget
+      class="groupbyColumnsInput"
+      v-model="editedStep.groups"
+      name="Group rows by..."
+      :options="columnNames"
+      placeholder="Add columns"
+      data-path=".groups"
+      :errors="errors"
+      :available-variables="availableVariables"
+      :variable-delimiters="variableDelimiters"
```

Comment:
Either use dashes everywhere or camelCase like L7, but maybe not both in the same template?
(We should have an eslint rule for this :p)

Reply:
Haha sorry I copied this from https://github.com/ToucanToco/weaverbird/blob/5fd8b5a862dfb4f4a4f1b79a8f645a4db9b44e61/src/components/stepforms/AggregateStepForm.vue#L31 

I can't override the prop names of `ListWidget` without changing everything and I don't think I can use kebab-case for variable names in javascript ? So I'm not really sure about what to do here